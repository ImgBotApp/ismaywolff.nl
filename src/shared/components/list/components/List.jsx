import styled from 'styled-components'

// prettier-ignore
const List = styled.ul`
  padding-left: 0;
`

export default List
