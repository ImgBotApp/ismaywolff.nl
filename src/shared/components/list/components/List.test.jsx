import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import List from './List'

describe('<List />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<List />)
    expect(wrapper).toMatchSnapshot()
  })
})
