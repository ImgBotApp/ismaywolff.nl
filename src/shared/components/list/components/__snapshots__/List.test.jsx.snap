// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> renders correctly 1`] = `
.c0 {
  padding-left: 0;
}

<ul
  className="c0"
/>
`;
